customersupport/customersupport_tests.py

```diff
@@ -0,0 +1,153 @@
+import unittest
+import requests
+import requests_mock
+from config import SALES_URL, HR_URL
+from customersupport.wrappers import sales
+from customersupport.models import Employee
+from customersupport.wrappers import hr, mocked_responses
+from customersupport.wrappers import hr as hr_wrapper
+
+
+class Stubber:
```

Comment:
Rename `Stubber` class to match the other ones, each specific to an API endpoint.